mlir/lib/Analysis/SliceAnalysis.cpp

```diff
@@ -109,6 +109,8 @@ static LogicalResult getBackwardSliceImpl(Operation *op,
         if (parentOp->getNumRegions() == 1 &&
             llvm::hasSingleElement(parentOp->getRegion(0).getBlocks())) {
           return getBackwardSliceImpl(parentOp, backwardSlice, options);
+        } else {
+          return failure();
```

Comment:
Can this be tested?

Reply:
We're looking for an operation which  is not IsolatedFromAbove, have multiple regions or a region with multiple blocks?